### PR TITLE
Implement hamburger menu

### PR DESCRIPTION
**Pull request by sauloperez**
The last piece of the new site.

Screenshot from 2019-09-23 16-53-32
Screenshot from 2019-09-23 16-53-50

### CODE REVIEW

**Comment by sauloperez** [comment]
I take that as an approval @sseerrggii :trollface:

File: assets/js/index.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+(function() {
+
+  var hamburger = {
```
Comment (andresgutgon) [inline assets/js/index.js:3]: Is this valid JavaScript for all the browsers?

Comment (sauloperez) [reply assets/js/index.js:3]: no idea. You tell me :trollface:

Comment (andresgutgon) [reply assets/js/index.js:3]: What are the supported browsers for this site?

Comment (andresgutgon) [reply assets/js/index.js:3]: Any way I think is this and I think it's ok https://caniuse.com/#search=variable%20block forget it :)

**Comment by sauloperez** [comment]
I take your review as an approval @andresgutgon

**Pull request merged**